spec.md

```diff
@@ -0,0 +1,555 @@
+﻿# Data Provenance Metadata Version 1.0
+**Editor's Note:**
+This title was put in by editor and does not imply consensus by group. 
+It is just so there is text there
+
+**Editor's Note:**
+Editor needs to work out how to include image required (OASIS Logo) at begining
+
+## Committee Specification Draft 01
+
+## 06 May 2025
+
+**Editor's Note:**
+THe following "version links" will be filled out nearer to completion. Note for 'first' version, 
+"This" and "latest" will be the same, and "previous" will be "none" 
+
+### This version
+
+- \[ link to authoritative version of the published document \] (Authoritative)  
+- \[ links to one or more other versions of the published document (e.g., MD, PDF, Word, HTML, etc.) \] 
+
+
+### Previous version
+
+
+- \[ link to authoritative version of the published document \] (Authoritative)  
+- \[ links to one or more other versions of the published document (e.g., MD, PDF, Word, HTML, etc.) \] 
+
+
+### Latest version
+
+
+- \[ link to authoritative version of the published document \] (Authoritative)  
+- \[ links to one or more other versions of the published document (e.g., MD, PDF, Word, HTML, etc.) \] 
+
+
+
+### Technical Committee
+
+
+[Data Provenance Standard Technical Committee](https://groups.oasis-open.org/communities/tc-community-home2?CommunityKey=2c60b2cf-45d3-48cd-8594-0194f182b33d)
+
+
+### Chairs
+
+
+- Lisa Bobbitt, Cisco, lbobbitt@cisco.com
+- Bryan Bortnick, IBM, bortnick@us.ibm.com
+- Fotis Psallidas, Microsoft, Fotis.Psallidas@microsoft.com
+
+
+
+### Secretaries
+
+
+- Kristina Podnar, Data & Trust Alliance, kpodnar@dataandtrustalliance.org 
+
+
+### Editors
+
+
+- Stefan Hagen, individual, stefan@hagen.link

```

Comment:
Ibid (sort naturally):

- Duncan Sparrell, sFractal Consulting, duncan@sfractal.com
- Stefan Hagen, individual, stefan@hagen.link

Note: I already commented on the source at https://github.com/sparrell/dps/commit/52e810be52a45e72c1465ddc6e6e31d8ac1d7eab#diff-bc6661da34ecae62fbe724bb93fd69b91a7f81143f2683a81163231de7e3b545R62